test/test_files/patcher_data/samus_returns/starter_preset.json

```diff
@@ -1,702 +1,3375 @@
 {
     "starting_location": {
-        "scenario": "s000_surface",
-        "actor": "StartPoint0"
+      "scenario": "s000_surface",
+      "actor": "StartPoint0"
     },
     "starting_items": {
```

Comment:
It's missing a few things we explicitly set to be required by the schema of OSRR

Reply:
Aren't the extra fields handled by the patcher itself? Those fields were removed from the patcher inventory update at some point.

Reply:
Yeah, it was removed here https://github.com/randovania/open-samus-returns-rando/pull/79/files#diff-e75ec462766acda07074dd7aa8051b093052e93a1b6564968da9f50b9c31628cL59

I don't care that much how we do it but we need to agree on what we are doing. I removed it because the calculation of e.g. max energy was not correct if it is already present in the starting items.

We should force providing values for stuff which makes sense to be changed.
_MAX value definitely makes sense because it is the starting ammo + implicit mains.
Energy makes sense if we want to start with an arbitrary amount of health. Could remove it, make it implicit to 99 and just allow starting e-tanks (which the patcher already handles correctly by adding them to the max energy)
What does happen if we change the metroid count and total count? Is this just for the visual at the bottom right screen?

Reply:
I haven't messed with the Metroid values so I have no clue.